builder/builder.go

```diff
@@ -39,7 +39,7 @@ func NewBuilder(pm *procmanager.ProcManager, debug bool, workspaceDir string) *B
 // RunTask executes a Task.
 func (b *Builder) RunTask(ctx context.Context, task *graph.Task) error {
 	for _, network := range task.Networks {
-		log.Printf("Creating Docker network: %s\n", network.Name)
+		log.Printf("Creating Docker network: %s, driver: '%s'\n", network.Name, network.Driver)
```

Comment:
how about `"%+v", network`?

Reply:
Gonna leave as-is for now, we can add a `ToString` if we want to print out the full network contents later at some point. I'm not even sure how much value there is in printing out the driver.